web/pwa/components/history/Week.js

```diff
@@ -44,6 +44,22 @@ export function Week({
     selectedPeriodEnd,
     missionsInPeriod
   );
+  const missionsToDetail = React.useMemo(
+    () => missionsInPeriod.filter(mission => !mission.isHoliday),
+    [missionsInPeriod]
+  );
+
+  const hasWorkMissions = React.useMemo(
+    () => missionsInPeriod.filter(mission => !mission.isHoliday).length > 0,
+    [missionsInPeriod]
+  );
+  const kpis = React.useMemo(() => {
+    let allKpis = renderPeriodKpis(stats).filter(m => m.name !== "service");
+    if (hasWorkMissions) {
+      return allKpis;
+    }
+    return allKpis.filter(m => m.name === "offDays");
```

Comment:
:green_circle: Même remarque ici : 
```suggestion
    let allKpis = renderPeriodKpis(stats).filter(kpi => kpi.name !== "service");
    if (hasWorkMissions) {
      return allKpis;
    }
    return allKpis.filter(kpi => kpi.name === "offDays");
```